tika-parsers/src/main/resources/org/apache/tika/parser/captioning/tf/Im2txtRestDockerfile

```diff
@@ -46,7 +43,7 @@ RUN \
     wget https://raw.githubusercontent.com/apache/tika/master/tika-parsers/src/main/resources/org/apache/tika/parser/captioning/tf/caption_generator.py \
         -O caption_generator.py && \
 
-    wget https://raw.githubusercontent.com/apache/tika/master/tika-parsers/src/main/resources/org/apache/tika/parser/captioning/tf/im2txtapi.py \
+    wget https://raw.githubusercontent.com/ThejanW/tika/master/tika-parsers/src/main/resources/org/apache/tika/parser/captioning/tf/im2txtapi.py \
```

Comment:
Reminder this needs to be changed back....

Reply:
will do once, merged :+1: